frontend/src/components/Profile/MyFriends.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-card :loading="loading" class="ma-5" max-width="400">
+    <v-card-title class="white--text orange darken-4">
+      Friends
+
+      <v-spacer></v-spacer>
+
+      <v-btn color="white" class="text--primary" fab small>
+        <v-icon>mdi-plus</v-icon>
+      </v-btn>
+    </v-card-title>
+
+    <v-virtual-scroll :items="users" item-height="50" height="300" width="500">
+      <template v-slot:default="{ item }">
+        <v-list-item>
+          <v-list-item-content>
+            <v-list-item-title>{{ item.username }}</v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </template>
+    </v-virtual-scroll>
+  </v-card>
+</template>
+
+<script>
```

Comment:
```suggestion
<script lang="ts">
```

I think that we should mark the script as typescript so that we can take advantage of type hinting features. However, you won't be happy with the errors that appear when you do that

So we can perhaps handle that one view at a time in separate issues. What do you think?

Reply:
So I think separate issues.

Reply:
I opened #114